package.json

```
@@ -86,7 +86,7 @@
     "@bbc/psammead-copyright": "^3.0.5",
     "@bbc/psammead-detokeniser": "^1.0.0",
     "@bbc/psammead-embed-error": "^3.0.6",
-    "@bbc/psammead-episode-list": "0.1.0-alpha.22",
+    "@bbc/psammead-episode-list": "1.0.0",
```

Shouldn't this be `1.0.3` instead?

```suggestion
    "@bbc/psammead-episode-list": "1.0.3",
```

---

any reason we're not using the caret?

---

I can add caret + 1.0.3.

The PR is old af, I made it during the change freeze just to tie up a bit of Psammead work.